### PR TITLE
add contract navigation

### PR DESCRIPTION
**Pull request by julia-andersson**
## [APP-XXX]

- Insurances navigation

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Contracts/Sources/View/UpcomingChangesScreen.swift [hunk 1]
```diff
@@ -40,16 +42,15 @@ struct UpcomingChangesScreen: View {
...
+                            contractsNavigationVm.isChatPresented = true
                         } content: {
                             hText(L10n.openChat)
                         }
 
                     }
                     hSection {
                         hButton.LargeButton(type: .ghost) {
-                            store.send(.contractDetailNavigationAction(action: .dismissUpcomingChanges))
+                            //                            store.send(.contractDetailNavigationAction(action: .dismissUpcomingChanges))
```
Comment (sladan-hedvig) [inline Projects/Contracts/Sources/View/UpcomingChangesScreen.swift:53]: Lets start writing //TODO: 'comment'
everywhere we have comments

File: Projects/Contracts/Sources/View/UpcomingChangesScreen.swift [hunk 2]
```diff
@@ -82,21 +83,21 @@ struct UpcomingChangesScreen_Previews: PreviewProvider {
...
-            options: [.largeNavigationBar]
-        ) { action in
-            if case .contractDetailNavigationAction(action: .dismissUpcomingChanges) = action {
-                PopJourney()
-            }
-        }
-        .configureTitle(L10n.InsuranceDetails.updateDetailsSheetTitle)
-    }
-}
+//extension UpcomingChangesScreen {
```
Comment (sladan-hedvig) [inline Projects/Contracts/Sources/View/UpcomingChangesScreen.swift:86]: Delete if we don't need it anymore/ we added it

File: Projects/EditCoInsured/Sources/Journeys/EditCoInsuredJourney.swift [hunk 3]
```diff
@@ -69,16 +67,17 @@ public struct EditCoInsuredViewJourney: View {
...
 
     func openInsuredPeopleScreen(with config: InsuredPeopleConfig) -> some View {
         let store: EditCoInsuredStore = globalPresentableStoreContainer.get()
+        store.coInsuredViewModel.config = config
         return InsuredPeopleScreen(vm: store.coInsuredViewModel, intentVm: store.intentViewModel)
     }
 }
 
-//
 //public class EditCoInsuredJourney {
```
Comment (sladan-hedvig) [inline Projects/EditCoInsured/Sources/Journeys/EditCoInsuredJourney.swift:81]: Lets clean comments as much as possible

**Review by sladan-hedvig — COMMENTED** [review]
Nothing huge, just some thinks that we might need to take care for the future checks.
Mostly start using "//TODO: "

File: Projects/Contracts/Sources/EditContract.swift [hunk 4]
```diff
@@ -4,12 +4,14 @@ import hCore
 import hCoreUI
 import hGraphQL
 
-struct EditContract: View {
+public struct EditContract: View {
```
[the remark this answers is not in the document]
Comment (julia-andersson) [reply Projects/Contracts/Sources/EditContract.swift:7]: Yes you're right, since we moved the logic back to contracts 👍

**Review by sladan-hedvig — APPROVED** [review]
🥳

**Pull request merged**